src/__tests__/u-chartStorage.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const fs = require("fs")
+const chartStorage = require('../lib/chartStorage')
+require('@testing-library/jest-dom')
+
+function initDomFromFiles(htmlPath, jsPath) {
+	const html = fs.readFileSync(htmlPath, 'utf8')
+	document.open()
+	document.write(html)
+	document.close()
+	jest.isolateModules(function() {
+		require(jsPath)
+	})

```

Comment:
This function has the same indentation difference as in `u-generateChartImg.js`. It looks like you're using 4 spaces normally, but this function uses tabs instead. Small potatoes though :)